DemoServer/Controllers/Demos/CompareExchange/IndexCompareExchange/metadata.json

```diff
@@ -0,0 +1,161 @@
+{
+    "Slug": "index-compare-exchange",
+    "SourceFileName": "IndexCompareExchangeController.cs",
+    "Title": "Index CmpXchg Values",
+    "DescriptionLines": [
+        "**CmpXchg Values can be loaded & indexed** within the index definition.",
+        "",
+        "Any change in the indexed CmpXchg Values will trigger re-indexing."
```

Comment:
any change in the indexed ones, or any change of any cmpXchg value?

Reply:
as I understand - reindexing will occur for any change made to the indexed cmpxchg values - those that are related/associated with the key that was used in the Map definition